addons/web_editor/static/src/js/common/ace-dialog.js

```diff
@@ -0,0 +1,109 @@
+/** @odoo-module **/
+
+import { loadBundle } from "@web/core/assets";
+import Dialog from 'web.Dialog';
+import core, { _t } from 'web.core';
+
+/**
+ * Render an code editor in the given node.
+ * @param {DOMElement} node where the editor will be rendered
+ * @param {String | undefined} content text of the editor
+ * @returns {Object} the ace editor instance
+ */
+export function renderCodeEditor(node, content) {
```

Comment:
Nice that you are adding a doc :+1: 
- Add a blank line between the description and the first `@param`
- DOMElement is not a type
- Instead of `{String | undefined} content`, write `{string} [content]` (optional param and built-in type params should be lowercase)